### PR TITLE
Remove GRPC middleware

### PR DESCRIPTION
**Pull request by emersonwood**
Removes outdated / redundant GRPC middleware between client and platform.

This makes future API changes between the two a lot less painful.

### CODE REVIEW

File: app/commands/describe_fact.go [hunk 1]
```diff
@@ -62,14 +58,15 @@ func describeFact(ctx *cli.Context) error {
 		return errors.New("Please specify a properly namespaced fact, ie,\nlingo describe-fact codelingo/go/func_decl")
```
Comment (clairscent) [inline app/commands/describe_fact.go]: error strings should not be capitalized or end with punctuation or a newline

File: app/commands/review/review.go [hunk 2]
```diff
@@ -1,26 +1,19 @@
...
 	"github.com/codelingo/lingo/service"
 	grpcclient "github.com/codelingo/lingo/service/grpc"
-	"github.com/codelingo/lingo/service/server"
 	"github.com/codelingo/platform/flow/rpc/client"
 	flow "github.com/codelingo/platform/flow/rpc/flowengine"
-
-	"context"
-
 	"github.com/juju/errors"
 )
```
Comment (clairscent) [inline app/commands/review/review.go]: exported function RequestReview should have comment or be unexported

File: app/commands/review/review.go [hunk 3]
```diff
@@ -162,106 +155,6 @@ l:
 	return confirmedIssues, nil
 }
```
Comment (clairscent) [inline app/commands/review/review.go]: exported function NewRange should have comment or be unexported

File: app/commands/review/review.go [hunk 4]
```diff
@@ -1,26 +1,19 @@
 package review
```
Comment (clairscent) [inline app/commands/review/review.go]: should have a package comment, unless it's in another file for this package

File: app/commands/tooling.go [hunk 5]
```diff
@@ -81,14 +77,15 @@ func listFacts(ctx *cli.Context) error {
 		return errors.New("Please specify a properly namespaced lexicon, ie,\nlingo lexicons list-facts codelingo/go")
```
Comment (clairscent) [inline app/commands/tooling.go]: error strings should not be capitalized or end with punctuation or a newline

File: service/service.go [hunk 6]
```diff
@@ -1,87 +1,21 @@
 package service
```
Comment (clairscent) [inline service/service.go]: should have a package comment, unless it's in another file for this package

File: service/service.go [hunk 7]
```diff
@@ -268,479 +196,98 @@ func runQuery(ctx context.Context, client codelingo.CodeLingoClient, queryc chan
...
 	if err != nil {
 		return nil, err
 	}
-
-	r := reply.(map[string][]string)
-
-	return r, nil
-
+	return reply.Lexicons, nil
 }
```
Comment (clairscent) [inline service/service.go]: exported function ListFacts should have comment or be unexported

File: service/service.go [hunk 8]
```diff
@@ -268,479 +196,98 @@ func runQuery(ctx context.Context, client codelingo.CodeLingoClient, queryc chan
 	return resultc, nil
 }
```
Comment (clairscent) [inline service/service.go]: exported function ListLexicons should have comment or be unexported

File: service/service.go [hunk 9]
```diff
@@ -268,479 +196,98 @@ func runQuery(ctx context.Context, client codelingo.CodeLingoClient, queryc chan
...
-	if err != nil {
-		return err
-	}
-
-	err = cPub.Publish("", buf)
-	if err != nil {
-		return err
-	}
-	return nil
 }
```
Comment (clairscent) [inline service/service.go]: exported function DescribeFact should have comment or be unexported

File: service/service.go [hunk 10]
```diff
@@ -1,87 +1,21 @@
...
-	grpcclient "github.com/codelingo/lingo/service/grpc"
-
 	"github.com/codelingo/lingo/service/grpc/codelingo"
-	// kitot "github.com/codelingo/kit/tracing/opentracing"
-
-	"github.com/codelingo/lingo/service/server"
+	"github.com/juju/errors"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
 )
```
Comment (clairscent) [inline service/service.go]: exported function Review should have comment or be unexported

File: service/service.go [hunk 11]
```diff
@@ -122,10 +56,6 @@ func Review(ctx context.Context, req *codelingo.ReviewRequest) (chan *codelingo.
 	return issuec, nil
 }
```
Comment (clairscent) [inline service/service.go]: exported function Query should have comment or be unexported

File: service/service.go [hunk 12]
```diff
@@ -268,479 +196,98 @@ func runQuery(ctx context.Context, client codelingo.CodeLingoClient, queryc chan
...
-			"describefact":        buildEndpoint(tracer, "describefact", instances, describeFactFactory, randomSeed, logger),
-			"latestclientversion": buildEndpoint(tracer, "latestclientversion", instances, latestClientVersionFactory, randomSeed, logger),
-		},
-	}, nil
-}
-
-type grpcOptions struct {
-	tls bool
+	return reply, nil
 }
```
Comment (clairscent) [inline service/service.go]: exported function LatestClientVersion should have comment or be unexported

File: service/service.go [hunk 13]
```diff
@@ -268,479 +196,98 @@ func runQuery(ctx context.Context, client codelingo.CodeLingoClient, queryc chan
...
-func (c client) LatestClientVersion() (string, error) {
-	request := codelingo.LatestClientVersionRequest{}
-	reply, err := c.endpoints["latestclientversion"](c.Context, request)
-	if err != nil {
-		return "", err
-	}
-	r := reply.(*codelingo.LatestClientVersionReply)
-	return r.Version, nil
+	return reply, nil
 }
```
Comment (clairscent) [inline service/service.go]: exported function PathsFromOffset should have comment or be unexported

File: app/commands/review/review.go [hunk 14]
```diff
@@ -162,106 +155,6 @@ l:
 	return confirmedIssues, nil
 }
 
-func showIngestProgress(progressc server.Ingestc, messagec server.Messagec) error {
```
Comment (waigani) [inline app/commands/review/review.go]: It's a shame to see this go, but it's out of scope to bring it back.

**Pull request merged**